src/pages/system-management/dict/list/Index.js

```diff
@@ -0,0 +1,389 @@
+import PageHeaderWrapper from '@/components/PageHeaderWrapper';
+import StandardTable from '@/components/StandardTable';
+import { Button, Card, DatePicker, Divider, Dropdown, Form, Icon, Input, Menu, Modal, Popconfirm } from 'antd';
+import { connect } from 'dva';
+import React, { Fragment, PureComponent } from 'react';
+import styles from '../../../List/TableList.less';
+const { MonthPicker, RangePicker, WeekPicker } = DatePicker;
+const FormItem = Form.Item;
+
+const CreateForm = Form.create()(props => {
+    const { modalVisible, form, handleAdd, handleModalVisible } = props;
+    const okHandle = () => {
+        form.validateFields((err, fieldsValue) => {
+            if (err) return;
+            handleAdd(fieldsValue);
+        });
+    };
+    return (
+        <Modal
+            destroyOnClose
+            title="新建字典"
+            visible={modalVisible}
+            onOk={okHandle}
+            onCancel={() => handleModalVisible()}
```

Comment:
Codacy found an issue: [Expected indentation of 10 space characters but found 12.](https://app.codacy.com/manual/afc163/ant-design-pro/pullRequest?prid=6006324)